robottelo/ui/locators/menu.py

```diff
@@ -4,6 +4,16 @@
 from selenium.webdriver.common.by import By
 from .model import LocatorDict
 
+# This is the old menu container xpath, was failing in upstream
```

Comment:
'old|new' should suffice our needs, don't need to go in that details. 'Keep way' is not the same as 'forbidden' or something. Just a recommendation from my side